tpchgen/src/generators.rs

```diff
@@ -144,6 +152,13 @@ impl<'a> Iterator for NationGeneratorIterator<'a> {
 }
 
 /// The REGION table
+/// The Display trait is implemented to format the line item data as a string
```

Comment:
To keep the same "pattern" with previous comment can we have a new line in between the top level comment and the next one.
```suggestion

/// The Display trait is implemented to format the line item data as a string
```